### PR TITLE
[13.0][MIG]pos_order_to_sale_order, standard migration

### PR DESCRIPTION
**Pull request by manuelregidor**
Standard migration

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍🏻

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
@legalsylvain @ivantodorovich  could you review/merge, please?

THX!

File: pos_order_to_sale_order/__manifest__.py [hunk 1]
```diff
@@ -14,4 +14,5 @@
     "website": "https://github.com/OCA/pos",
     "data": ["views/view_pos_config.xml", "views/assets.xml"],
     "qweb": ["static/src/xml/pos_order_to_sale_order.xml"],
+    "installable": True,
```
Comment (ivantodorovich) [inline pos_order_to_sale_order/__manifest__.py:17]: Not necessary but ok :)

**Review by ivantodorovich — APPROVED** [review]
LGTM :+1: 

It'd be good to add some tests. Not blocking as there weren't any before, though it'd be nice
Let me know if you can handle that otherwise we can just merge

**Comment by HaraldPanten** [comment]
Hi @ivantodorovich 

We can not afford these tests in the next few weeks. But we do not discard to make an [IMP] PR in a future.

Could you merge, meanwhile?

THX so much 👍

**Pull request merged**